pump/server.go

```diff
@@ -58,6 +58,10 @@ var (
 	detectDrainerCheckpointInterval = 10 * time.Minute
 	// GlobalConfig is global config of pump
 	GlobalConfig *globalConfig
+	// functions
+	getPdClient         = util.GetPdClient
```

Comment:
you can use getPdClientFn

Reply:
Revised

Reply:
you can change all, the comment "functions" is redundant

Reply:
Revised again